modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/SearchBar.kt

```diff
@@ -155,7 +169,7 @@ private fun SearchBarDarkPreview() {
 @Composable
 private fun SearchBarPreview() {
     Column(modifier = Modifier.padding(8.dp)) {
-        SearchBar("Material", onTextChanged = {}, modifier = Modifier.padding(bottom = 8.dp))
+        SearchBar("Materiaxdzsdasd asdasdas sdadsa asdadsa asdasdas asdsada asdadsl", onTextChanged = {}, modifier = Modifier.padding(bottom = 8.dp))
```

Comment:
I laughed 😆

Reply:
Who has never done this? 😆

Reply:
Oops! 🙈 reverted in https://github.com/Automattic/pocket-casts-android/pull/2077/commits/0812d506d87c1ff417c1b8b22b9fddc6c6fd77f3